### PR TITLE
Add ep.30

### PR DESCRIPTION
**Pull request by PercyDan54**

### CODE REVIEW

File: S3/English/ASS_30.ass [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:02:38.54,0:02:39.65,Default,,0,0,0,,I'm here with condolences
+Dialogue: 0,0:02:41.98,0:02:42.93,Default,,0,0,0,,Is Momo ok
+Dialogue: 0,0:02:43.21,0:02:44.84,Default,,0,0,0,,Hasn't drink any alcohol for two days
+Dialogue: 0,0:02:46.06,0:02:47.48,Default,,0,0,0,,I'll put juice and desserts here
+Dialogue: 0,0:02:49.30,0:02:50.03,Default,,0,0,0,,Who is this
+Dialogue: 0,0:02:51.31,0:02:52.23,Default,,0,0,0,,Look familiar
+Dialogue: 0,0:02:53.18,0:02:54.98,Default,,0,0,0,,That's the photo of me when I was an idol
+Dialogue: 0,0:02:55.18,0:02:55.86,Default,,0,0,0,,Idol?
+Dialogue: 0,0:02:57.71,0:02:58.43,Default,,0,0,0,,Hello everyone
+Dialogue: 0,0:02:58.64,0:03:00.59,Default,,0,0,0,,We are
```
Comment (PercyDan54) [inline S3/English/ASS_30.ass:78]: Need a name for this (毛毛团子)

File: S3/English/ASS_30.ass [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:03:03.16,0:03:04.46,Default,,0,0,0,,Then why did you give up
+Dialogue: 0,0:03:05.12,0:03:06.46,Default,,0,0,0,,It's too much stress to be an idol
+Dialogue: 0,0:03:07.05,0:03:08.91,Default,,0,0,0,,But it's mainly my habit is difficult to change
+Dialogue: 0,0:03:09.34,0:03:11.28,Default,,0,0,0,,And I was photographed on the headlines
+Dialogue: 0,0:03:14.64,0:03:15.55,Default,,0,0,0,,How did they recognize you
+Dialogue: 0,0:03:15.55,0:03:16.79,Default,,0,0,0,,The reporter is so powerful
```
Comment (PercyDan54) [inline S3/English/ASS_30.ass:84]: Need a better tl

File: S3/English/ASS_30.ass [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:03:47.78,0:03:48.49,Default,,0,0,0,,Cheers
+Dialogue: 0,0:03:50.55,0:03:51.52,Default,,0,0,0,,How did this happen
+Dialogue: 0,0:03:53.13,0:03:54.00,Default,,0,0,0,,It's the drink I bought
+Dialogue: 0,0:03:54.78,0:03:55.77,Default,,0,0,0,,It contains alcohol?
+Dialogue: 0,0:03:56.88,0:03:58.45,Default,,0,0,0,,We're all drinking 
+Dialogue: 0,0:03:59.61,0:04:01.90,Default,,0,0,0,,Now it's your turn
+Dialogue: 0,0:04:07.10,0:04:11.44,Default,,0,0,0,,With the effort from Lynn and all members of 1031
+Dialogue: 0,0:04:11.74,0:04:12.64,Default,,0,0,0,,Momo
+Dialogue: 0,0:04:12.85,0:04:16.04,Default,,0,0,0,,Finally quit drinking after a year
+Dialogue: 0,0:04:16.68,0:04:19.89,Default,,0,0,0,,But the
```
Comment (PercyDan54) [inline S3/English/ASS_30.ass:108]: How to tl “地狱般的"

**Review by PercyDan54 — COMMENTED** [review]
Will do a second pass for timings soon

File: S3/English/ASS_30.ass [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:01:33.45,0:01:34.40,Default,,0,0,0,,Lynn
+Dialogue: 0,0:01:34.65,0:01:35.75,Default,,0,0,0,,She's here again
+Dialogue: 0,0:01:37.40,0:01:37.80,Default,,0,0,0,,You
+Dialogue: 0,0:01:38.78,0:01:39.28,Default,,0,0,0,,Stay back
+Dialogue: 0,0:01:40.00,0:01:41.60,Default,,0,0,0,,One step ahead and I'm done with you
+Dialogue: 0,0:01:42.24,0:01:42.70,Default,,0,0,0,,Why
+Dialogue: 0,0:01:42.70,0:01:43.30,Default,,0,0,0,,Stay back
+Dialogue: 0,0:01:44.02,0:01:44.90,Default,,0,0,0,,Let's talk outside
+Dialogue: 0,0:01:45.52,0:01:47.47,Default,,0,0,0,,I will send a bill of damaged items to your email
+Dialogue: 0,0:01:47.77,0:01:48.85,Default,,0,0,0,,Anyway before you quit drinking
```
Comment (Hrshl7) [inline S3/English/ASS_30.ass:46]: add a comma after "anyway"

File: S3/English/ASS_30.ass [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:02:55.10,0:02:55.80,Default,,0,0,0,,Idol?
+Dialogue: 0,0:02:57.70,0:02:58.30,Default,,0,0,0,,Hello everyone
+Dialogue: 0,0:02:58.60,0:03:00.59,Default,,0,0,0,,We are
+Dialogue: 0,0:03:03.16,0:03:04.46,Default,,0,0,0,,Then why did you give up
+Dialogue: 0,0:03:05.12,0:03:06.40,Default,,0,0,0,,It's too stressful to be an idol
+Dialogue: 0,0:03:07.05,0:03:08.91,Default,,0,0,0,,But it's mainly my habit is difficult to change
+Dialogue: 0,0:03:09.30,0:03:11.20,Default,,0,0,0,,And I was photographed on the headlines
+Dialogue: 0,0:03:14.64,0:03:15.45,Default,,0,0,0,,How did they recognize you
+Dialogue: 0,0:03:15.45,0:03:16.79,Default,,0,0,0,,The reporter is so powerful
+Dialogue: 0,0:03:17.93,0:03:19.29,Default,,0,0,0,,Then I wanted to change a environment
```
Comment (Hrshl7) [inline S3/English/ASS_30.ass:85]: "**of** environment" maybe?

File: S3/English/ASS_30.ass [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+Dialogue: 0,0:03:43.40,0:03:44.75,Default,,0,0,0,,Why Lily isn't watching her
+Dialogue: 0,0:03:45.72,0:03:46.19,Default,,0,0,0,,Lily
+Dialogue: 0,0:03:46.66,0:03:47.30,Default,,0,0,0,,Together
+Dialogue: 0,0:03:47.78,0:03:48.49,Default,,0,0,0,,Cheers
+Dialogue: 0,0:03:50.55,0:03:51.52,Default,,0,0,0,,How did this happen
+Dialogue: 0,0:03:53.13,0:03:54.00,Default,,0,0,0,,It's the drink I bought
+Dialogue: 0,0:03:54.78,0:03:55.77,Default,,0,0,0,,It contains alcohol?
+Dialogue: 0,0:03:56.88,0:03:58.45,Default,,0,0,0,,We're all drinking
+Dialogue: 0,0:03:59.61,0:04:01.90,Default,,0,0,0,,Now it's your turn
+Dialogue: 0,0:04:07.10,0:04:11.35,Default,,0,0,0,,With the effort from Lynn and all members of 1031
```
Comment (Hrshl7) [inline S3/English/ASS_30.ass:105]: "all the" or "every" would fit better

File: S3/English/ASS_30.ass (hunk 2 above)
Comment (Hrshl7) [reply S3/English/ASS_30.ass]: "The reporter is so good"? (maybe "talented")

File: S3/English/ASS_30.ass [hunk 7]
```diff
@@ -26,32 +26,32 @@ Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
...
-Dialogue: 0,0:01:28.42,0:01:29.44,Default,,0,0,0,,Someone broke into here?
+Dialogue: 0,0:01:28.35,0:01:29.30,Default,,0,0,0,,Someone broke into here?
 Dialogue: 0,0:01:31.10,0:01:31.71,Default,,0,0,0,,Anyone
 Dialogue: 0,0:01:33.45,0:01:34.40,Default,,0,0,0,,Lynn
 Dialogue: 0,0:01:34.65,0:01:35.75,Default,,0,0,0,,She's here again
-Dialogue: 0,0:01:37.41,0:01:38.00,Default,,0,0,0,,You
+Dialogue: 0,0:01:37.40,0:01:37.80,Default,,0,0,0,,You
 Dialogue: 0,0:01:38.78,0:01:39.28,Default,,0,0,0,,Stay back
-Dialogue: 0,0:01:40.33,0:01:41.60,Default,,0,0,0,,One step ahead and I'm done with you
+Dialogue: 0,0:01:40.00,0:01:41.60,Default,,0,0,0,,One step ahead and I'm done with you
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:41]: One more step and I won't hesitate

File: S3/English/ASS_30.ass [hunk 8]
```diff
@@ -26,32 +26,32 @@ Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
...
-Dialogue: 0,0:01:45.52,0:01:47.47,Default,,0,0,0,,I will send a bill of damaged stuff to your email
-Dialogue: 0,0:01:47.77,0:01:48.88,Default,,0,0,0,,Anyway before you quit drinking
-Dialogue: 0,0:01:49.01,0:01:49.95,Default,,0,0,0,,Don't come to 1031 again
-Dialogue: 0,0:01:52.51,0:01:52.98,Default,,0,0,0,,What
+Dialogue: 0,0:01:45.52,0:01:47.47,Default,,0,0,0,,I will send a bill of damaged items to your email
+Dialogue: 0,0:01:47.77,0:01:48.85,Default,,0,0,0,,Anyway before you quit drinking
+Dialogue: 0,0:01:49.00,0:01:49.95,Default,,0,0,0,,Don't come to 1031 again
+Dialogue: 0,0:01:52.45,0:01:52.98,Default,,0,0,0,,What
 Dialogue: 0,0:01:53.16,0:01:54.50,Default,,0,0,0,,I broke all of that?!
 Dialogue: 0,0:01:58.65,0:01:59.25,Default,,0,0,0,,Above all
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:50]: ...as s a result of what I just said

File: S3/English/ASS_30.ass [hunk 9]
```diff
@@ -26,32 +26,32 @@ Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
...
-Dialogue: 0,0:02:00.14,0:02:01.70,Default,,0,0,0,,Lily will live with you these days
-Dialogue: 0,0:02:02.27,0:02:03.02,Default,,0,0,0,,As a supervisor
-Dialogue: 0,0:02:03.22,0:02:04.61,Default,,0,0,0,,Hello, Momo
-Dialogue: 0,0:02:05.84,0:02:07.14,Default,,0,0,0,,So b
+Dialogue: 0,0:02:00.10,0:02:01.70,Default,,0,0,0,,Lily will live with you these days
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:51]: Lily will live with you for the time being

File: S3/English/ASS_30.ass [hunk 10]
```diff
@@ -26,32 +26,32 @@ Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
...
+Dialogue: 0,0:02:02.15,0:02:03.02,Default,,0,0,0,,As a supervisor
+Dialogue: 0,0:02:03.20,0:02:04.61,Default,,0,0,0,,Hello, Momo
+Dialogue: 0,0:02:05.84,0:02:07.14,Default,,0,0,0,,So thank you
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:54]: Thank you for your trouble, then

File: S3/English/ASS_30.ass [hunk 11]
```diff
@@ -26,32 +26,32 @@ Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,10
...
 Dialogue: 0,0:02:10.65,0:02:12.64,Default,,0,0,0,,Lily should be asleep now
 Dialogue: 0,0:02:13.28,0:02:14.22,Default,,0,0,0,,Now
 Dialogue: 0,0:02:15.38,0:02:16.30,Default,,0,0,0,,Lily you underestimated me so much
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:57]: You have underestimated me, Lily

File: S3/English/ASS_30.ass [hunk 12]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:20.80,0:02:22.10,Default,,0,0,0,,Don't drink alcohol
 Dialogue: 0,0:02:22.10,0:02:23.00,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:25.07,0:02:25.97,Default,,0,0,0,,Don't drink alcohol
-Dialogue: 0,0:02:26.20,0:02:27.50,Default,,0,0,0,,Lily is watching you
+Dialogue: 0,0:02:26.20,0:02:27.40,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:28.15,0:02:29.63,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:29.68,0:02:30.79,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:30.90,0:02:32.12,Default,,0,0,0,,Lily is watching you
 Dialogue: 0,0:02:32.81,0:02:35.04,Default,,0,0,0,,How did you find me
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:67]: How do you keep finding me

File: S3/English/ASS_30.ass [hunk 13]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
-Dialogue: 0,0:02:37.44,0:02:38.40,Default,,0,0,0,,Lily, Momo
-Dialogue: 0,0:02:38.54,0:02:39.65,Default,,0,0,0,,I'm here with condolences
+Dialogue: 0,0:02:37.35,0:02:38.40,Default,,0,0,0,,Lily, Momo
+Dialogue: 0,0:02:38.54,0:02:39.55,Default,,0,0,0,,I'm here with condolences
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:69]: I'm here with gifts

File: S3/English/ASS_30.ass [hunk 14]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:02:41.98,0:02:42.93,Default,,0,0,0,,Is Momo ok
 Dialogue: 0,0:02:43.21,0:02:44.84,Default,,0,0,0,,Hasn't drink any alcohol for two days
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:71]: She hasn't drunk any alcohol for two days

File: S3/English/ASS_30.ass [hunk 15]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
-Dialogue: 0,0:02:46.06,0:02:47.48,Default,,0,0,0,,I'll put juice and desserts here
-Dialogue: 0,0:02:49.30,0:02:50.03,Default,,0,0,0,,Who is this
-Dialogue: 0,0:02:51.31,0:02:52.23,Default,,0,0,0,,Look familiar
-Dialogue: 0,0:02:53.18,0:02:54.98,Default,,0,0,0,,That's the photo of me when I was an idol
-Dialogue: 0,0:02:55.18,0:02:55.86,Default,,0,0,0,,Idol?
-Dialogue: 0,0:02:57.71,0:02:58.43,Default,,0,0,0,,Hello everyone
-Dialogue: 0,0:02:58.64,0:03:00.59,Default,,0,0,0,,We are
+Dialogue: 0,0:02:46.06,0:02:47.48,Default,,0,0,0,,I'll put juice and desserts on the desk
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:72]: I'll put the juice and desserts on the desk

File: S3/English/ASS_30.ass [hunk 16]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
+Dialogue: 0,0:02:49.30,0:02:49.80,Default,,0,0,0,,Who is this
+Dialogue: 0,0:02:51.15,0:02:52.20,Default,,0,0,0,,Looks familiar
+Dialogue: 0,0:02:53.18,0:02:54.70,Default,,0,0,0,,That's the photo of me when I was an idol
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:75]: That's a photo of me when I was an idol

File: S3/English/ASS_30.ass [hunk 17]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
+Dialogue: 0,0:02:55.10,0:02:55.80,Default,,0,0,0,,Idol?
+Dialogue: 0,0:02:57.70,0:02:58.30,Default,,0,0,0,,Hello everyone
+Dialogue: 0,0:02:58.60,0:03:00.59,Default,,0,0,0,,We are
 Dialogue: 0,0:03:03.16,0:03:04.46,Default,,0,0,0,,Then why did you give up
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:79]: Then why did you quit

File: S3/English/ASS_30.ass [hunk 18]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
+Dialogue: 0,0:03:05.12,0:03:06.40,Default,,0,0,0,,It's too stressful to be an idol
 Dialogue: 0,0:03:07.05,0:03:08.91,Default,,0,0,0,,But it's mainly my habit is difficult to change
-Dialogue: 0,0:03:09.34,0:03:11.28,Default,,0,0,0,,And I was photographed on the headlines
-Dialogue: 0,0:03:14.64,0:03:15.55,Default,,0,0,0,,How did they recognize you
-Dialogue: 0,0:03:15.55,0:03:16.79,Default,,0,0,0,,The reporter is so powerful
-Dialogue: 0,0:03:17.93,0:03:19.29,Default,,0,0,0,,So I wanted to change a environment
+Dialogue: 0,0:03:09.30,0:03:11.20,Default,,0,0,0,,And I was photographed on the headlines
+Dialogue: 0,0:03:14.64,0:03:15.45,Default,,0,0,0,,How did they recognize you
+Dialogue: 0,0:03:15.45,0:03:16.79,Default,,0,0,0,,The reporter is so powerful
+Dialogue: 0,0:03:17.93,0:03:19.29,Default,,0,0,0,,Then I wanted to change a environment
```
Comment (Torikuso) [inline S3/English/ASS_30.ass]: Then, I wanted to change my environment

File: S3/English/ASS_30.ass [hunk 19]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:03:19.44,0:03:20.10,Default,,0,0,0,,And start fresh
-Dialogue: 0,0:03:21.02,0:03:21.79,Default,,0,0,0,,So I was here
+Dialogue: 0,0:03:20.95,0:03:21.79,Default,,0,0,0,,So I was here
 Dialogue: 0,0:03:22.26,0:03:23.69,Default,,0,0,0,,You are so brave to start fresh
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:88]: You are so brave to start from zero again

File: S3/English/ASS_30.ass [hunk 20]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:03:23.92,0:03:24.75,Default,,0,0,0,,Cheer up, Momo
```
Comment (Torikuso) [inline S3/English/ASS_30.ass]: I'm rooting for you, Momo

File: S3/English/ASS_30.ass [hunk 21]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:03:25.13,0:03:26.45,Default,,0,0,0,,You will finally quit drinking
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:90]: You will successfully quit drinking

File: S3/English/ASS_30.ass [hunk 22]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
-Dialogue: 0,0:03:27.65,0:03:28.17,Default,,0,0,0,,Thank you
+Dialogue: 0,0:03:27.55,0:03:28.17,Default,,0,0,0,,Thank you
 Dialogue: 0,0:03:33.07,0:03:34.39,Default,,0,0,0,,Come on
 Dialogue: 0,0:03:35.10,0:03:37.31,Default,,0,0,0,,Let's drink the whole box
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:93]: Today we finish the whole crate

File: S3/English/ASS_30.ass [hunk 23]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
-Dialogue: 0,0:03:37.77,0:03:39.51,Default,,0,0,0,,Drink until you're drunk
-Dialogue: 0,0:03:42.40,0:03:43.47,Default,,0,0,0,,Where's Lily
-Dialogue: 0,0:03:43.82,0:03:44.96,Default,,0,0,0,,Why Lily isn't watching her
+Dialogue: 0,0:03:37.75,0:03:39.35,Default,,0,0,0,,Drink until you're drunk
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:94]: No resting until you're drunk

File: S3/English/ASS_30.ass [hunk 24]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
+Dialogue: 0,0:03:42.60,0:03:43.15,Default,,0,0,0,,Where's Lily
+Dialogue: 0,0:03:43.40,0:03:44.75,Default,,0,0,0,,Why Lily isn't watching her
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:96]: Why isn't Lily watching her

File: S3/English/ASS_30.ass [hunk 25]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:03:45.72,0:03:46.19,Default,,0,0,0,,Lily
-Dialogue: 0,0:03:46.66,0:03:47.52,Default,,0,0,0,,Together
+Dialogue: 0,0:03:46.66,0:03:47.30,Default,,0,0,0,,Together
 Dialogue: 0,0:03:47.78,0:03:48.49,Default,,0,0,0,,Cheers
 Dialogue: 0,0:03:50.55,0:03:51.52,Default,,0,0,0,,How did this happen
 Dialogue: 0,0:03:53.13,0:03:54.00,Default,,0,0,0,,It's the drink I bought
 Dialogue: 0,0:03:54.78,0:03:55.77,Default,,0,0,0,,It contains alcohol?
-Dialogue: 0,0:03:56.88,0:03:58.45,Default,,0,0,0,,We're all drinking 
+Dialogue: 0,0:03:56.88,0:03:58.45,Default,,0,0,0,,We're all drinking
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:103]: We're both drinking

File: S3/English/ASS_30.ass [hunk 26]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
 Dialogue: 0,0:03:59.61,0:04:01.90,Default,,0,0,0,,Now it's your turn
-Dialogue: 0,0:04:07.10,0:04:11.44,Default,,0,0,0,,With the effort from Lynn and all members of 1031
-Dialogue: 0,0:04:11.74,0:04:12.64,Default,,0,0,0,,Momo
-Dialogue: 0,0:04:12.85,0:04:16.04,Default,,0,0,0,,Finally quit drinking after a year
-Dialogue: 0,0:04:16.68,0:04:19.89,Default,,0,0,0,,But the 
-Dialogue: 0,0:04:20.38,0:04:23.77,Default,,0,0,0,,Nobody ever wanted to talk about it
+Dialogue: 0,0:04:07.10,0:04:11.35,Default,,0,0,0,,With the effort from Lynn and all members of 1031
+Dialogue: 0,0:04:11.95,0:04:12.45,Default,,0,0,0,,Momo
+Dialogue: 0,0:04:12.85,0:04:16.00,Default,,0,0,0,,Finally succeeded in quiting drinking after a year
+Dialogue: 0,0:04:16.68,0:04:19.85,Default,,0,0,0,,But the
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:108]: But the hellish process it took to get there

File: S3/English/ASS_30.ass [hunk 27]
```diff
@@ -60,50 +60,50 @@ Dialogue: 0,0:02:17.55,0:02:18.46,Default,,0,0,0,,Lily is watching you
...
+Dialogue: 0,0:04:20.25,0:04:23.77,Default,,0,0,0,,Nobody ever wanted to talk about it again
```
Comment (Torikuso) [inline S3/English/ASS_30.ass:109]: Is something nobody ever wanted to talk about again

File: S3/English/ASS_30.ass (hunk 1 above)
Comment (Torikuso) [reply S3/English/ASS_30.ass]: I suggest "Fluffy Dangos" or "Furry Dangos". The group's name was never mentioned in the manhua as far as I remember, so we're gonna have to make an original TL here.

File: S3/English/ASS_30.ass (hunk 2 above)
Comment (Torikuso) [reply S3/English/ASS_30.ass]: Perhaps "The reporter has a really good eye" ?

**Pull request merged**